config/samples/resources/accesscontextmanagerserviceperimeterresource/accesscontextmanager_v1beta1_accesscontextmanagerserviceperimeterresource.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: accesscontextmanager.cnrm.cloud.google.com/v1beta1
+kind: AccessContextManagerServicePerimeterResource
+metadata:
+  name: accesscontextmanagerserviceperimeterresource-sample
+spec:
+  perimeterNameRef:
+    # Referencing a resource name leads to recursive reference and KCC does not support the feature for now.
+    # Please use external reference instead.
+    # Replace "${ACCESS_POLICY_NUMBER}" with the numeric ID for your Access Policy
+    # Replace "${PERIMETER_SHORT_NAME}" with the name for the Service Perimeter to add this resource to
+    external: "accessPolicies/${ACCESS_POLICY_NUMBER}/servicePerimeters/${PERIMETER_SHORT_NAME}"
+  projectRef:
```

Comment:
`resourceRef`?

Could you clean up the yamls and run the tests locally to ensure they all work as expected?

Reply:
Thanks! updated!